packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -817,6 +817,7 @@ describeFullCompat("Detached Container", (getTestObjectProvider) => {
 				if (runtimeMessage === false) {
 					return;
 				}
+				// TODO: message.contents is now unknown. what to do?
 				const envelope = message.contents.contents.content;
```

Comment:
@agarwal-navin Any ideas for how to fix this code?

Reply:
Navin is currently OOF, but I think it would be something like:
```
const envelope = (
	(message.contents as { contents: unknown }).contents as {
		content: unknown;
	}
).content as any;
```